commons-text/src/main/java/org/jdbi/v3/commonstext/UseStringSubstitutorTemplateEngine.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.commonstext;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.jdbi.v3.commonstext.internal.UseStringSubstitutorTemplateEngineImpl;
+import org.jdbi.v3.sqlobject.config.ConfiguringAnnotation;
+
+@Retention(RetentionPolicy.RUNTIME)
+@ConfiguringAnnotation(UseStringSubstitutorTemplateEngineImpl.class)
+@Target({ElementType.TYPE, ElementType.METHOD})
+public @interface UseStringSubstitutorTemplateEngine {
+    /**
+     * default: <code>${</code>
+     */
+    String prefix() default StringSubstitutorTemplateEngine.DEFAULT_PREFIX;
```

Comment:
This isn't what I meant--sorry for the misunderstanding.

What I meant is, when you run the build, does javadoc output the default value for the attribute, even though it is a reference to a constant in another class? I ran a build on your branch, and the answer is yes, it does:

```
Element Detail

    prefix

    public abstract String prefix

    default: ${     <-- your javadoc comment

    Default:        <-- generated out of the box by javadoc
        "${"
```

So we can remove those comments, since Javadoc is automatically documenting the default value for us.

Reply:
ok, reverted